migrations/2_deploy_contracts.js

```diff
@@ -59,72 +53,125 @@ module.exports = async function(deployer, network) {
-        "ERC20 Address": erc20MinterPauserInstance.address,
-        "ERC20LockRelease Address": erc20LockReleaseInstance.address,
-        "ERC721 Address": erc721MinterBurnerPauserInstance.address,
-        "ERC20Handler Address": erc20HandlerInstance.address,
-        "ERC20LockReleaseHandler Address": erc20HandlerInstance.address,
-        "ERC721Handler Address": erc721HandlerInstance.address,
-        "GenericHandler Address": genericHandlerInstance.address,
-        "FeeRouterContract Address": feeRouterInstance.address,
-        "BasicFeeHandler Address": basicFeeHandlerInstance.address,
-        "FeeHandlerWithOracle Address": feeHandlerWithOracleInstance.address,
-        "ERC20 resourceID": currentNetworkConfig.erc20ResourceID,
-        "Generic resourceID": currentNetworkConfig.genericResourceID,
-        "ERC721 resourceID": currentNetworkConfig.erc721ResourceID,
-        "ERCLockRelease20 resourceID": currentNetworkConfig.erc20LockReleaseResourceID
-    });
-
-    /* setup contracts */
-    const tokenAmount = process.env.TOKEN_AMOUNT;
-    const feeOracleAddress = process.env.FEE_ORACLE_ADDRESS;
-    const feeHandlerWithOracleGasUsed = process.env.FHWO_GAS_USED;
-    const feeHandlerWithOracleFeePercentage = process.env.FHWO_FEE_PERCENTAGE; // e.g. 500 means 5%
-    const basicFeeHandlerFee = Ethers.utils.parseEther(process.env.BFH_FEE);
-
-    // setup ERC20
-    await bridgeInstance.adminSetResource(erc20HandlerInstance.address, currentNetworkConfig.erc20ResourceID, erc20MinterPauserInstance.address);
-    await bridgeInstance.adminSetResource(erc20HandlerInstance.address, currentNetworkConfig.erc20LockReleaseResourceID, erc20LockReleaseInstance.address);
-    await erc20MinterPauserInstance.grantRole(await erc20MinterPauserInstance.MINTER_ROLE(), erc20HandlerInstance.address);
-    await erc20MinterPauserInstance.mint(deployerAddress, tokenAmount);
-    await erc20MinterPauserInstance.mint(erc20HandlerInstance.address, tokenAmount);
-    await erc20MinterPauserInstance.approve(erc20HandlerInstance.address, tokenAmount);
-    await bridgeInstance.adminSetBurnable(erc20HandlerInstance.address, erc20MinterPauserInstance.address);
-
-    // setup ERC721
-    await bridgeInstance.adminSetResource(erc721HandlerInstance.address, currentNetworkConfig.erc721ResourceID, erc721MinterBurnerPauserInstance.address);
-    await erc721MinterBurnerPauserInstance.grantRole(await erc20MinterPauserInstance.MINTER_ROLE(), erc721HandlerInstance.address);
-    await bridgeInstance.adminSetBurnable(erc721HandlerInstance.address, erc721MinterBurnerPauserInstance.address);
-
-    // setup generic
-    await bridgeInstance.adminSetGenericResource(genericHandlerInstance.address, currentNetworkConfig.genericResourceID, TestStoreInstance.address, Helpers.blankFunctionSig, Helpers.blankFunctionDepositorOffset, Helpers.getFunctionSignature(TestStoreInstance, 'store'));
+    // deploy safe contracts
+    await deployer.deploy(ERC20SafeContract);
```

Comment:
Should not we then put deploys under the if statement on lines 90 and 139?

Reply:
Actually, these are extended and shouldn't be deployed at all.
Will remove those lines.